modules/core/src/main/scala/gem/config/DynamicConfig.scala

```diff
@@ -7,22 +7,22 @@ import java.time.Duration
 
 sealed trait SmartGcalKey
 
-sealed abstract class InstrumentConfig extends Product with Serializable {
+sealed abstract class DynamicConfig extends Product with Serializable {
+
+  type I <: Instrument with Singleton
```

Comment:
I know, or thought I knew, what a `Singleton` is but what is it here?  I couldn't find the documentation for the trait.